R/read-brd.R

```diff
@@ -6,8 +6,8 @@
 #' @inheritParams tsdbr::ts_create_db
 #' @export
 ts_read_brd <- function(file = "brd.csv", utc_offset = -8L) {
-  check_string(file)
-  check_scalar(utc_offset, c(-12L, 14L))
+  chk_string(file)
+  chk_scalar(utc_offset, c(-12L, 14L))
```

Comment:
`chk_scalar` only checks if scalar need a separate `check_values()` for integer and range